source/test/TestConfig.cpp

```diff
@@ -0,0 +1,72 @@
+// This is a derivative work. originally part of the LLVM Project.
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Copyright (c) 2023 Klemens D. Morgenstern
+//
+
+#include "TestConfig.hpp"
+#include <mrdox/Error.hpp>
+#include <llvm/Support/FileSystem.h>
+#include <llvm/Support/Path.h>
+#include <llvm/Support/YAMLParser.h>
+#include <llvm/Support/YAMLTraits.h>
+
+
+template<>
+struct llvm::yaml::MappingTraits<
+    clang::mrdox::TestConfig>
```

Comment:
Wait a minute I think there's some confusion here, the tests are supposed to use the same YML configuration as the tool. Not a whole new YML configuration.

Reply:
It's a different config object.